README.md

```diff
@@ -1,24 +1,3 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+This work-in-progress app allows playgroups to keep game scores and players’ results over time, as well as track analytics of deck performance. The workflow is designed to simulate working within a BDD agile development team, each piece of code is peer reviewed and tests are included in the CI. As the project nears the MVP stage I will add more details here about how to run the project locally. The end goal is to deploy and host this on a dedicated website for public use. 
```

Comment:
> tracking analytics on deck performance

?

Reply:
We'll eventually be able to answer questions such as : which deck wins the most, which player, playing someone else's deck, wins the most? etc